script.js

```diff
@@ -660,36 +638,18 @@ function handleRentStrikeLayer(geoJson) {
 function handleNationsLayer(geojson) {
   const layerOptions = {
     style: feature => {
-      const { passed } = feature.properties;
-      // If law is passed
-      if (passed) {
-        return {
-          color: '#4dac26',
-          fillColor: '#b8e186',
-          fillOpacity: fillOpacity,
-          weight: strokeWeight,
-        };
-      } 
-      // If not yet passed
-      if (!passed) {
-        return {
-          color: '#d01c8b',
-          fillColor: '#f1b6da',
-          fillOpacity: fillOpacity,
-          weight: strokeWeight,
-        };
-      }
-      // If no value for passed
       return {
-        stroke: false,
-        fill: false,
+        color: strokeColorScale[feature.properties.range] || colorNoData,
+        fillColor: fillColorScale[feature.properties.range] || colorNoData,
+        fillOpacity: fillOpacity,
+        weight: strokeWeight
       };
     },
   };
 
   // Create the Leaflet layer for the nations data
   const nationsLayer = L.geoJson(geojson, layerOptions);
-  
+
   nationsLayer.bindPopup(function (layer) {
     const { name_en, passed } = layer.feature.properties;
```

Comment:
Do we still want to show "passed" in the infowindow? If we don't need it, we should modify the template, remove it from the queries, and we don't need to pass the "passed" info to the template.

Reply:
At this point we can consolidate the `info-window-template-nation` and `info-window-template`. We should decide what the content of that window should be but at this point they are basically the same, so we should use the same template and make sure were passing the correct props.

Reply:
Regarding `passed`, if we want to keep it, it's more for the data team to maintain. Also I had the text for the policy strength in the popup before, maybe we want to move back to that, so all these layers would show the policy strength in the popup instead of the state of `passed`

Reply:
Thanks @river-honer for catching that. Agree that consolidating the template would be ideal, and I also liked your idea of having the text for the score display in the infowindow. I think it’s a good UX that helps reaffirm the meaning of the color. We could also give the text a background color that’s the color associated with the score which would help as well reaffirm this as well.

Reply:
Oh that's a good idea.

Reply:
I made notes for what changes need to be made in a comment at the bottom.